### PR TITLE
docs: io.cozy.files thumbnails links validity

### PR DESCRIPTION
**Pull request by Crash--**
Thanks for contributing to this documentation. If you added a doctype, please be sure that it is present 

- [x] in the `Readme.md` index page
- [x] in the `toc.yml` page

### CODE REVIEW

**Review by JF-Cozy — APPROVED** [review]
LGTM 👍

File: docs/io.cozy.files.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ The attributes of a file are:
 
 It also has a relationship with its `parent` in the JSON-API representation.
 
-For an `image`, there are 3 links to thumbnails: `small`, `medium`, and `large`.
+For an `image`, there are 3 links to thumbnails: `small`, `medium`, and `large`. 
+Theses thumbnails' links are valid for 10 minutes.
```
Comment (ptbrowne) [inline docs/io.cozy.files.md:57]:
```suggestion
These thumbnails' links are valid for 10 minutes. 
```

Comment (ptbrowne) [reply docs/io.cozy.files.md:57]: After 10 minutes, what happens ? The links send back 404 ou 500 ?

Comment (Crash--) [reply docs/io.cozy.files.md:57]: The secret contained in the link is not valid anymore, I assume I will have a "wrong token" https://github.com/cozy/cozy-stack/blob/master/web/files/files.go#L815, so a 400.

Comment (ptbrowne) [reply docs/io.cozy.files.md]: Thanks for the addition 👍

**Review by ptbrowne — APPROVED** [review]

**Pull request merged**